template/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: Netlify Release
+on:
+  push:
+    branches: ['main', 'master']
+  workflow_dispatch:
+    inputs:
+      deploy-msg:
+        description: 'Deploy Message (visible in Netlify)'
+        required: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Should we add this GitHub actions as the first step:
https://github.com/styfle/cancel-workflow-action

Then it can stop the previous run when we run another one 🤔